index.js

```diff
@@ -65,45 +65,41 @@ module.exports = {
     const trueEmberVersion = emberVersion.match(/\d+\.\d+\.\d+/)[0];
 
     let options = {
-      envFlags: {
+      debugTools: {
+        isDebug: true,
```

Comment:
Not entirely sure what the implications of `true` vs `false` here or if this should be driven from the environment that the Ember CLI is being run in